### DISCUSSION TITLE
Discord community

### DISCUSSION THREAD

**Discussion by JesseVelden** [adelsz/pgtyped#449] [Ideas]
Would it be nice to have a Discord community for quick questions, and discussing + planning of new features? @adelsz

**Comment by adelsz** [comment]
Agree, I have created a [gitter](https://gitter.im/pgtyped/community) for now instead. Had too many issues with discord in the past and not a fan of their business model and closed source nature. We can reconsider this again as the community grows.

**Reply by JesseVelden** [reply]
Fair enough, it's a long time I used gitter extensively. I hope people will join :)

**Reply by delight** [reply]
I think this awesome project would benefit greatly from a official discord server!

**Reply by JesseVelden** [reply]
I also think an active community can help the project further, and I tend to agree that Discord is having the most momentum/ now almost the main thing that is being used for open source projects and its communities. We use Gitter for over a year now but there are not any interesting discussions being held.

That being said, I also understand Adel's point of view: https://twitter.com/adelssk/status/1684597252713512960. Unfortunately the network effects are holding alternative/ more open platforms like Gitter back. It's the same reason that people write on X/ twitter because there are already lots of other people on it.

**Reply by delight** [reply]
I totally agree .. it is far from perfect. There is even more shortcomings then just the locked-in content - but looking at it practically it is the best way to build a community - I feel like nowadays it is almost expected to have a discord server.

**Reply by JesseVelden** [reply]
Do you want to reconsider it @adelsz?
